### ISSUE TITLE
Different visualization

### ISSUE THREAD

**Issue by stheid** [mwhamgenomics/PySplit#7]
Do you think there is any chance to implement a overlay-like interface that is not in the console but just draws over the game? I am not so much into UI stuff, but perhaps someone else has ideas of how this can be done easily

**Comment by mwhamgenomics** [comment]
Yeah, I'm not that much of a UI programmer, not sure exactly how this would be done. So far I've been able to have the terminal running the timer in the foreground because the game has been on a separate console/PC.

If the game is on the same machine, the client would need to listen for inputs/keyboard events in the background. Apparently LiveSplit can do this with global hotkeys, not sure if the current curses client could do the same - possibly OS-dependent? I imagine a fully integrated overlay would involve writing a new client using the OS's graphical APIs. Apparently full-screen is complicated.

For a first step, it should be easy enough to write up a spec for implementing a new client - inputs it should accept, server API calls it should make, etc.

**Comment by stheid** [comment]
yeah, having an clear interface that a client has to implement would be definetly a nice step towards that direction. i am currently too busy to tinker arround with UI dev.
